robots/panther/custom_config.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# If a particular robot requires any extra custom configuration steps, they should be included in this file.
+
+CONFIG_FILES_PATH="/etc/husarion/robot_configs/panther"
+
+# Time server config
+cp ${CONFIG_FILES_PATH}/files/chrony.conf /etc/chrony/chrony.conf
+cp ${CONFIG_FILES_PATH}/files/chrony.service /lib/systemd/system/chrony.service
+cp ${CONFIG_FILES_PATH}/files/hwclock-set /lib/udev/hwclock-set
+
+# Setup envs
+echo "ROS_IP=10.15.20.3" >> /etc/environment
+echo "ROS_MASTER_URI=http://10.15.20.2:11311" >> /etc/environment
+echo "SBC_NAME_FANCY='Intel NUC'" >> /etc/environment
```

Comment:
Shouldn't the `SBC_NAME_FANCY` be set in the image? I think that these configs shouldn't have hardcoded sbc

Reply:
Fixed:
https://github.com/husarion/husarion-os/pull/5